File: src/net/t3kt/tctrl/schema/params/OtherParamSchema.java
package net.t3kt.tctrl.schema.params;

import net.t3kt.tctrl.schema.TctrlSchemaProto.ParamSpec;

public final class OtherParamSchema extends SingleParamSchema {
    OtherParamSchema(ParamSpec spec) {
        super(spec);
    }

    public String getOtherType() {
        return spec.getOtherType();
    }
}

File: win/TctrlCore/Schema/Params/OtherParamSchema.cs
using System;
using Tctrl.Schema;

namespace Tctrl.Core.Schema.Params
{
    public sealed class OtherParamSchema : SingleParamSchema
    {
        public OtherParamSchema(ParamSpec spec) : base(spec)
        {
        }

        public String OtherType => Spec.OtherType;
    }
}